pyproject.toml

```diff
@@ -86,11 +86,6 @@ show_error_codes = true
 warn_return_any = true
 warn_unused_ignores = true
 
-# Prevent mypy from trigger the missing import error for our written packages
```

Comment:
I was wrong, this is needed :ugh: but probably you can merge this into the tool.mypy block up above, no need to override.